frontend/stylesheets/all.scss

```diff
@@ -5,7 +5,7 @@
 @import "foundation";
 @import "foundation-icons";
 @import "foundation/functions";
-@import "highlight.css";
+// @import "highlight.css";
```

Comment:
We should implement the highlight in another way bc right now this file has ruby code.

```ruby
<%= Rouge::Themes::ThankfulEyes.render(:scope => '.highlight') %>
```